api/models/Bed.js

```diff
@@ -0,0 +1,52 @@
+/* global BedEvent*/
+'use strict';
+
+var LinkingModels = require('sails-linking-models');
+var ModelHelpers = require('../lib/ModelHelpers');
+
+const model = {
+  autoCreatedAt: true,
+  autoUpdatedAt: true,
+  schema: true,
+  attributes: {
+    centre: {
+      model: "centres",
+      required: true
+    },
+    bed_ref: {
+      type: 'string',
+      required: true
+    },
+    gender: {
+      type: 'string',
+      enum: ['male', 'female'],
+      required: false,
+      defultsTo: null
+    }
+  },
+
+  getGender: (gender) => {
```

Comment:
consider the existing `normalizeGender` we have in the `detainee` model, refactoring that out to somewhere?